src/destructors.md

```diff
@@ -57,48 +62,85 @@ core::mem::forget(partial_move.1);
 
 ## Drop scopes
 
+r[destructor.scope]
```

Comment:
```suggestion
r[destructors.scope]
```

The document goes back and forth between the plural and singular.  Seems these should all be plural.